src/recommendations/troubleshooting.md

```diff
@@ -23,6 +23,7 @@ For issues related to the Catalog SaaS Export module:
        "magento/module-catalog-url-rewrite-data-exporter"
        "magento/module-configurable-product-data-exporter"
        "magento/module-data-exporter"
+       "magento/module-data-services" (use the latest version. Required >=3.0.5)
```

Comment:
I'm still not sure about the following assumption

>  The `saas-export` metapackage installs the following modules
> ...
> "magento/module-data-services"
> ...

From my understanding  `saas-export` and `data-services` are separate packages that are part of `product-recommendations` metapackage (see p.2 of [installation guide](https://devdocs.magento.com/recommendations/install.html#install))

But I may be wrong.

Reply:
@rogyar , your notice is correct. When you run 

```
composer require magento/product-recommendations
```

composer will install

- magento/module-data-services
- magento/module-product-recommendations
- magento/module-product-recommendations-layout
- magento/product-recommendations

Modules `magento/module-data-services` and `magento/module-data-services` should be installed separately.

Reply:
Hi @atwixfirster.

Let me explain in more examples. Please read the very first item of the [list](https://devdocs.magento.com/recommendations/troubleshooting.html#catalog-saas-export-module) which says:

> The saas-export metapackage installs the **following modules, all of which must be enabled**
> ...

After our modification in this PR we bring the following info:

- magento/module-data-services is a part of `saas-export` 
- magento/module-data-services MUST be enabled

which might be not 100% correct.

Reply:
@rossbrandon can I get your thoughts on this?

Reply:
`composer require magento/product-recommendations` will install the following packages:
`magento/services-connector`
`magento/module-services-id`
`magento/module-data-services`
`magento/saas-export`

`magento/product-recommendations` should be the entry point rather than installing the other packages separately. 

Once installed, `composer update magento/product-recommendations --with-dependencies` will update the packages that `magento/product-recommendations` depends on as well.

Reply:
@atwixfirster, @rogyar thank you for your feedback! The goal of this troubleshooting topic is to discuss issues with the Catalog SaaS Export module which does not include the data-services module. As @rossbrandon mentioned above, the product-recommendations module will install and handle these dependencies. While there are parts of the modules, such as the environment ID, which are introduced in one module and used in another, we want the documentation to reflect the recommended install process :`composer require magento/product-recommendations`.